src/dsymbol/conversion/package.d

```diff
@@ -103,6 +103,8 @@ class AutocompleteParser : Parser
 {
 	override BlockStatement parseBlockStatement()
 	{
+		if (!currentIs(tok!"{"))
```

Comment:
then after main fix segfault could happen here, for example when contracts were being written (incomplete at EOF)